packages/fxa-auth-server/lib/senders/emails/storybook-email.ts

```diff
@@ -9,9 +9,22 @@ export interface StorybookEmailArgs {
   variables: Record<string, any>;
 }
 
+export const commonArgs: StorybookEmailArgs['variables'] = {
+  acceptLanguage: 'en-US',
+  androidUrl:
+    'https://accounts-static.cdn.mozilla.net/product-icons/google-play.png',
+  iosUrl:
+    'https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png',
+  link: 'http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
+  privacyUrl:
+    'https://www.mozilla.org/privacy?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-privacy',
+  supportUrl:
+    'https://support.mozilla.org/kb/im-having-problems-with-my-firefox-account?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-support',
+};
+
 export const storybookEmail = ({
   template,
-  layout,
+  layout = 'fxa',
   doc,
```

Comment:
I think we should  add `acceptLanguage = 'en-US'` here like you gave `layout` the default value rather than include it in that `commonArgs` const since that's going to be the case across all of our templates in Storybook. `androidUrl`, `iosUrl`, etc. look cadReminder specific and probably shouldn't live in this file - we can rename that to something like `cadReminderCommonVars` and move that constant into `cadReminderFirst`, and then export it so you can import it in `cadReminderSecond`.

If we allow `layout` and `acceptLanguage` to have a default var, you can remove both of those in lowRecoveryCodes ([here](https://github.com/mozilla/fxa/pull/10092/files#diff-15a720ead2be22b837ac545709a374ad40147dd71ae3cacc3ed7ceec9449736cR21) and [here](https://github.com/mozilla/fxa/pull/10092/files#diff-15a720ead2be22b837ac545709a374ad40147dd71ae3cacc3ed7ceec9449736cR30)) and you won't have to reference those in other stories unless you need to override them. 👍